library/blob.c

```diff
@@ -442,6 +442,44 @@ static void eblob_dc_to_wc(const struct eblob_disk_control *dc, struct eblob_wri
 }
 
 /**
+ * Checks whether index and data disk control structures are the same.
+ */
+static int eblob_index_data_mismatch(const struct eblob_base_ctl *bctl,
+		const struct eblob_disk_control *dc,
+		const struct eblob_disk_control *data_dc)
+{
+	if (memcmp(&data_dc, dc, sizeof(struct eblob_disk_control))) {
```

Comment:
I suppose that `&` should be removed because currently it compares `const struct eblob_disk_control **` with `const struct eblob_disk_control *`.